stable/datadog/templates/daemonset.yaml

```diff
@@ -44,8 +44,10 @@ spec:
         {{- end }}
         {{- if and .Values.systemProbe.enabled .Values.daemonset.useDedicatedContainers }}
         container.apparmor.security.beta.kubernetes.io/system-probe: {{ .Values.systemProbe.apparmor }}
+        {{- if .Values.systemProbe.seccompHardened }}
```

Comment:
Sorry for being inconsistent on my side, I know I've suggested using a boolean for this, but I thought it'll be consistent if we could match the pattern for apparmor setup one line above. i.e. using `{{ .Values.systemProbe.seccomp }}` for the config value, and make the default value be `localhost/system-probe`. 
 We can always attach the existing seccomp but not use it. What do you think?

Reply:
Would the goal be to allow the user to provide its own seccomp profile ?
What would be the expected behaviour when the user specifies `systemProbe.seccomp = localhost/custom` ?
In that case, we should skip the copying of the `system-probe` secccomp profile to the host filesystem and try to use the one provided by the user ?

Reply:
@shang-wang I've just made the change so that we can specify any arbitrary `seccomp` profile.
If and only if the user let the default value (which is `localhost/system-probe`, the seccomp profile will be created.

Reply:
Well, that's the idea, but maybe we address this in another PR, to allow user to copy their own seccomp profile. But changing from boolean to a string as name can already provide the ability to select some other seccomp profiles like `runtime/default`, `docker/default`, etc, the docs are here: https://kubernetes.io/docs/concepts/policy/pod-security-policy/#seccomp. Does that make sense?

Reply:
I think the way it is now in the PR is quite flexible:
— If the user doesn’t override the `systemProbe.seccomp` value, it has the default value `localhost/system-probe` and that seccomp is created on-the-fly, like it is the case today with `master`.
— If the user wants to completely disable the seccomp sandboxing for `system-probe` for debugging purpose, he can override `systemProbe.seccomp` to either `runtime/default` or `false`.
— If the user has already deployed a seccomp profile in advance and he want to use it, he can just put its name in `systemProbe.seccomp`.